tools/cockpit.spec

```diff
@@ -358,6 +358,7 @@ Provides: %{name}-subscriptions = %{version}-%{release}
 Requires: subscription-manager >= 1.13
 Provides: %{name}-networkmanager = %{version}-%{release}
 Requires: NetworkManager
+Requires: NetworkManager-team
```

Comment:
Out of interest, why isn't this part of the standard NetworkManager code? 

In addition, I think this is missing a change further down in the spec file under the ```%package networkmanager```. It should probably have ```Suggests``` or ```Recommends``` style logic there.